### ISSUE TITLE
Resume from 1000000.pth pretrained checkpoint seems not working?

### ISSUE THREAD

**Issue by AmanuelHirpa** [naoto0804/pytorch-inpainting-with-partial-conv#29]

**Comment by greysou1** [comment]
Hi! I wanted to know how I can work with the pre-trained model, I just realized what a behemoth task it is to generate the data and train too.

**Comment by naoto0804** [comment]
> how I can work with the pre-trained model

Please see [here](https://github.com/naoto0804/pytorch-inpainting-with-partial-conv#test)

> I just realized what a behemoth task it is to generate the data and train too

Please refer to the original paper. For the mask, please refer [here](https://github.com/naoto0804/pytorch-inpainting-with-partial-conv/issues/20)
